### ISSUE TITLE
install instructions completely fail in 0.2.1

### ISSUE THREAD

**Issue by anarcat** [TobiasTheViking/flickrannex#15]
```
$ chmod +x git-annex-remote-flickr; sudo ln -sf `pwd`/git-annex-remote-flickr /usr/local/bin/git-annex-remote-flickr
chmod: cannot access 'git-annex-remote-flickr': No such file or directory
```

indeed, the file was completely removed in the last commit on this repository.

workaround: `git reset --hard HEAD^`

**Comment by lhunath** [comment]
Yes, something very strange happened in that 0.2.1 commit, a changelog was added mentioning a cache, but no cache was committed, rather the `git-annex-remote-flickr` was removed.  Presumably something got messed up in the attempt to commit 0.2.1 changes.
